Documentation/gettingstarted/k8s-install-rke.rst

```diff
@@ -0,0 +1,85 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _rke_install:
+
+********************************************
+Installation Using Rancher Kubernetes Engine
+********************************************
+
+This guide walks you through installation of Cilium on `Rancher Kubernetes Engine <https://rancher.com/products/rke/>`_,
+a CNCF-certified Kubernetes distribution that runs entirely within Docker containers.
+RKE solves the common frustration of installation complexity with Kubernetes by
+removing most host dependencies and presenting a stable path for deployment,
+upgrades, and rollbacks.
+
+Install a Cluster Using RKE
+===========================
+
+The first step is to install a cluster based on the `RKE Installation Guide <https://rancher.com/docs/rke/latest/en/installation/>`_.
+When creating the cluster, make sure to `change the default network plugin <https://rancher.com/docs/rke/latest/en/config-options/add-ons/network-plugins/custom-network-plugin-example/>`_
+in the config.yaml file.
+
+Change:
+
+.. code-block:: yaml
+
+  network:
+    options:
+      flannel_backend_type: "vxlan"
+    plugin: "canal"
+
+To:
+
+.. code-block:: yaml
+
+  network:
+    plugin: none
+
+Deploy Cilium
+=============
+
+.. tabs::
+
+    .. group-tab:: Installation via Helm v3
```

Comment:
Nitpick: The three tabs with “long” titles render on two lines, which is not a big issue but could maybe be improved:

![pic](https://user-images.githubusercontent.com/17001771/110926591-7fb7b180-831c-11eb-9ffe-6126f65956ba.png)

What if we had just `via Helm v3` etc.? We write again the full sentence `Install Cilium via helm install` inside the tab anyway.

```suggestion
.. tabs::

    .. group-tab:: Via Helm v3
```

Reply:
This was marked as resolved, but it doesn't appear to be resolved. Should be a simple change to improve the UI, can we just fix this up?